addons/web/static/src/core/dropdown/accordion_item.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="web.AccordionItem" owl="1">
+        <div class="o_accordion position-relative">
+            <button class="o_menu_item o_accordion_toggle dropdown-item" tabindex="0"
+                    t-att-class="{'selected': props.selected}"
+                    t-esc="props.description"
+                    t-on-click="() => state.open = !state.open"
+            />
+            <t t-if="state.open">
+                <div class="o_accordion_values ms-4 border-start" t-att-aria-expanded="state.open ? 'true' : 'false'">
```

Comment:
I suppose you can replace `t-att-aria-expanded="..."` with `aria-expanded="true"`, as this node is inside a `t-if="state.open"`